.github/ISSUE_TEMPLATE/module_testing.md

```diff
@@ -1,35 +0,0 @@
----
```

Comment:
why delete this? should we keep it for 26?

Reply:
We forgot about it and didn't use/update it for 24.1 or 25, so it's out of date and has the wrong label and wasn't used for the last two releases. 

I figured it's easiest to just clean it up and if we want to use a template for 26 we can bring it back and update it then.